bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="roku"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Roku Player Thing -->
+	<thing-type id="roku_player">
+		<label>Roku</label>
+		<description>
+			A Roku Streaming Media Player
+		</description>
+
+		<channels>
+			<channel id="activeApp" typeId="activeApp"/>
+			<channel id="button" typeId="button"/>
+			<channel id="playMode" typeId="playMode"/>
+			<channel id="timeElapsed" typeId="timeElapsed"/>
+			<channel id="timeTotal" typeId="timeTotal"/>
+		</channels>
+
+		<properties>
+			<property name="Model Name">unknown</property>
+			<property name="Model Number">unknown</property>
+			<property name="Device Location">unknown</property>
+			<property name="Serial Number">unknown</property>
+			<property name="Device Id">unknown</property>
+			<property name="Software Version">unknown</property>
+		</properties>
+
+		<representation-property>uuid</representation-property>
+
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<context>network-address</context>
+				<label>Host Name/IP Address</label>
+				<description>Host Name or IP Address of the Roku device</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="true">
+				<label>Port</label>
+				<description>Port for the ECP Connector of the Roku device</description>
+				<default>8060</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refresh" type="integer" min="10" required="false" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the Refresh Interval in Seconds</description>
+				<default>10</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+		</config-description>
```

Comment:
You can also use a config reference. So you don't have to repeat the config in both things.

Reply:
Changed to use a config reference. I had done this to be able to tweak the labels for standalone devices vs. TV.